libfwupdplugin/fu-quirks.c

```diff
@@ -174,9 +174,9 @@ fu_quirks_add_quirks_for_path (FuQuirks *self, XbBuilder *builder,
 		g_autoptr(XbBuilderSource) source = xb_builder_source_new ();
 
 		/* load from keyfile */
-		xb_builder_source_add_adapter (source, "text/plain,.quirk",
-					       fu_quirks_convert_quirk_to_xml_cb,
-					       NULL, NULL);
```

Comment:
You need to use `LIBXMLB_CHECK_VERSION` to avoid hard-depending on the very new symbol. Just fall back to the non-simple version.

Reply:
jinx, but you have a better proposed solution with the macro than the `#define`.

Reply:
It was like 12 seconds apart. Fixed. Thanks. I actually did not want the guard since I'm patching libxmlb 1.14 on my system and I overlooked this.